app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -1,176 +1,118 @@
 {{#if this.modelTaskInstance.isRunning}}
     <LoadingIndicator data-test-loading-indicator @dark={{true}} />
 {{else}}
-    <div local-class='table-filters'>
-        <div local-class='filter-controls'>
-            <div local-class='orcid-switch'>
-                <label for='switches-container' local-class='orcid-toggle-label'>{{t 'institutions.dashboard.users_list.has_orcid'}}</label>
-                <label local-class='switch'>
-                    <input id='switches-container' type='checkbox' checked={{this.hasOrcid}} {{on 'change' (fn this.toggleOrcidFilter (not this.hasOrcid))}}>
-                    <span local-class='slider round'></span>
-                </label>
-            </div>
-
-            <div local-class='department-filter'>
-                <PowerSelect
-                    @options={{this.departments}}
-                    @searchEnabled={{true}}
-                    @selected={{this.department}}
-                    @triggerClass={{local-class 'select'}}
-                    @search={{perform this.searchDepartment}}
-                    @onChange={{action this.onSelectChange}}
-                    as |department|
-                >
-                    {{department}}
-                </PowerSelect>
-            </div>
+    <div local-class='filter-container'>
+        <div local-class='orcid-switch'>
+            <label for='switches-container' local-class='orcid-toggle-label'>{{t 'institutions.dashboard.users_list.has_orcid'}}</label>
+            <label local-class='switch'>
+                <input id='switches-container' type='checkbox' checked={{this.hasOrcid}} {{on 'change' (fn this.toggleOrcidFilter (not this.hasOrcid))}}>
+                <span local-class='slider round'></span>
+            </label>
         </div>
+        <PowerSelect
+            @options={{this.departments}}
+            @searchEnabled={{true}}
+            @selected={{this.department}}
+            @triggerClass={{local-class 'select'}}
+            @search={{perform this.searchDepartment}}
+            @onChange={{this.onSelectChange}}
+            as |department|
+        >
+            {{department}}
+        </PowerSelect>
+        <BasicDropdown
+            @renderInPlace={{true}}
+            @onClose={{this.onDropdownClose}}
+            @preventScroll={{true}}
+            @eventType='mousedown'
+            local-class='dropdown'
+            as |dd|
+        >
+            <button type='button' local-class='dropdown-trigger' {{on 'click' dd.actions.toggle}}>
```

Comment:
I realize this styling is done to match what is in the mocks, but I think using `<Button>` here would help remove a lot of work from the css and prevent us from having an expanding list of "slightly different one-off styled" buttons, as we already have too many of those 😞 
```suggestion
            <Button local-class='dropdown-trigger' {{on 'click' dd.actions.toggle}}>
```